src/dfx/src/config/dfinity.rs

```diff
@@ -62,6 +63,7 @@ pub const DEFAULT_IC_GATEWAY_TRAILING_SLASH: &str = "https://ic0.app/";
 /// It only contains a type; everything else should be infered using the
 /// CanisterInfo type.
 #[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
Is there a way to configure serde to accept both kebab case and the default?  We already have `subnet_type`.

Reply:
It doesn't affect anything except the named fields of this particular struct. Unless you were talking about overall naming convention?

Reply:
Yes, bringing it up for overall consistency.

Reply:
Then it makes more sense not to include kebab at all.